docs/clone-sst.md

```diff
@@ -0,0 +1,190 @@
+
+### Donor and Joiner
+
+To enable the `clone` SST method, ensure the [`wsrep_sst_allowed_methods`](wsrep-system-index.md#wsrep_sst_allowed_methods) variable in the configuration file (`my.cnf`) includes the `clone` method for both the Donor and Joiner servers. This setting is essential for a successful State Snapshot Transfer (SST).
+
+Starting from Percona XtraDB Cluster 8.0.41, the default value of `wsrep_sst_allowed_methods` includes `clone`, which removes the need to configure this option manually in most cases.
+
+```ini
+[mysqld]
+wsrep_sst_allowed_methods = xtrabackup-v2,clone
+```
+
+### Joiner
+
+On the Joiner server, set the `wsrep_sst_method` variable to `clone` in the configuration file (`my.cnf`). This setting is the only accepted value for the Clone SST process.
+
+```ini
+[mysqld]
+wsrep_sst_method = clone
+```
+
+### Additional Information
+
+The `wsrep_sst_allowed_methods` and `wsrep_sst_method` variables are read-only and cannot be modified at runtime. You must set them in the configuration file before starting the server. Attempting to change these variables while the server is running will result in errors and may cause inconsistencies during node synchronization operations.
+
+For Percona XtraDB Cluster, any variables related to the SST mechanism, such as `wsrep_sst_allowed_methods` and `wsrep_sst_method`, must be defined before the  server startup to ensure proper synchronization.
+
+## Enable SSL for Clone SST
+
+To enable SSL for the Clone SST process, place the SSL certificates in a directory other than the data directory, as the clone process modifies this directory.
+
+We recommend explicitly setting the SSL certificates in the `my.cnf` file as follows:
+
+```ini
+[client]
+ssl-ca = /<path>/ca.pem
+ssl-cert = /<path>/client-cert.pem
+ssl-key = /<path>/client-key.pem
+
+[mysqld]
+ssl-ca = /<path>/ca.pem
+ssl-cert = /<path>/server-cert.pem
+ssl-key = /<path>/server-key.pem
+```
+
+Alternatively, you can configure the following SSL settings specifically for the Clone SST process on the Joiner:
+
+```ini
+[mysqld]
+clone_ssl_ca = /path/to/ca.pem
+clone_ssl_cert = /path/to/client-cert.pem
+clone_ssl_key = /path/to/client-key.pem
+```
+
+Ensure the `<path>` used is not the data directory to avoid conflicts during the Clone SST process.
+
+## Variables
+
+### Common variables
+
+State Snapshot Transfer (SST) in Galera Cluster relies on specific variables that control its configuration and behavior. YOu must set these variables appropriately to ensure seamless synchronization between nodes during the SST process. The following lists of the most commonly used variables and their purposes.
+
+* **`sst-idle-timeout`**:  
```

Comment:
I think we should be consistent with using dash or underscore. Let's stick to underscore, so sst_idle_timeout